easybuild/tools/options.py

```diff
@@ -207,6 +207,7 @@ def override_options(self):
                           "(e.g. --hide-deps=zlib,ncurses)", 'strlist', 'extend', None),
             'oldstyleconfig':   ("Look for and use the oldstyle configuration file.",
                                  None, 'store_true', True),
+            'only-module': ("Only generate module file", None, 'store_true', False),
```

Comment:
add the `and run sanity check` part (because that also happens,unless `--force` is used, right?)